README.md

```diff
@@ -700,6 +701,58 @@ The following props are passed to a custom field template component:
 
 > Note: you can only define a single field template for a form. If you need many, it's probably time to look at [custom fields](#custom-field-components) instead.
 
+### Array Template
+
+Similarly to the `FieldTemplate` you can use an `ArrayFieldTemplate` to customize how your
+arrays are rendered. This allows you to customize your array, and each element in the array.
+
+```jsx
+function ArrayFieldTemplate(props) {
+  return (
+    <div>
+      {props.items.map(element => element.children)}
+      {props.canAdd && <button onClick={props.onAddClick}></button>}
+    </div>
+  );
+}
+
+render((
+  <Form schema={schema}
+        ArrayFieldTemplate={ArrayFieldTemplate} />,
+), document.getElementById("app"));
+```
+
+Please see [customArray.js](https://github.com/mozilla-services/react-jsonschema-form/blob/master/playground/samples/customArray.js) for a better example.
+
+The following props are passed to each `ArrayFieldTemplate`:
+
+- `DescriptionField`: The generated `DescriptionField` (if you wanted to utilize it)
+- `TitleField`: The generated `TitleField` (if you wanted to utilize it)
+- `canAdd`: boolean whether new elements can be added to array
+- `className`: string
+- `disabled`: boolean
+- `idSchema`: Object
+- `items`: Object[] - Each of the items represent a child with properties described below.
+- `onAddClick`: (event) => void
+- `readonly`: boolean
+- `required`: boolean
+- `schema`: Object
+- `title`: string
+
+The following props are part of each element in `items`:
+
+- `children`: JSX.Element
+- `className`: string
+- `disabled`: boolean
+- `hasMoveDown`: boolean whether the element can be moved down
+- `hasMoveUp`: boolean whether the element can be moved up
+- `hasRemove`: boolean whether the element can be removed
+- `hasToolbar`: boolean
+- `index`: number
+- `onDropIndexClick`: (index) => void
+- `onReorderClick`: (index, newIndex) => void
+- `readonly`: boolean
```

Comment:
General remark for these two lists: it would be nice to have English sentences describing the purpose of each property, for consistency with the rest of the docs :)